testing/test_aux.out

```diff
@@ -58,8 +58,11 @@ RESULT pass security.passwords.ssh Default passwords have been changed.
 RESULT skip security.firmware Could not retrieve a firmware version with nmap. Check bacnet port.
 RESULT pass security.firmware version found: ?\xFF\xFF\x19,>u\x08\x00no
 RESULT pass connection.network.ntp_support Using NTPv4.
+RESULT fail connection.network.ntp_update Device clock not synchronized with local NTP server.
```

Comment:
Shouldn't one of these be pass?

Reply:
@grafnu The faux device/Travis seems to be returning a leap indicator of 3 regardless of the offset.